PPSTM_simple.py

```diff
@@ -143,7 +143,7 @@
     print("Importing positions of PP from the PP-AFM calculations. Path for the data:")
     path_pos="Q%1.2fK%1.2f/" %(Q,K)
     print(path_pos)
-    tip_r, lvec, nDim = GU.load_vec_field( path_pos+'PPpos' ,data_format=data_format)
+    tip_r, lvec, nDim, atomic_info, atomic_info_or_head = io.load_vec_field( path_pos+'PPpos' ,data_format=data_format)
```

Comment:
Wait here you have there twice: `atomic_info`  as you have `atomic_info_or_head`: it should be:
`tip_r, lvec, nDim, atomic_info_or_head = io. ...`

Reply:
Good catch! Fixed in new commit